k8s/demo/common/probate/probate-back-office.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: "hmctspublic.azurecr.io/probate/back-office:prod-67fd917f"
+      image: "hmctspublic.azurecr.io/probate/back-office:pr-1042-880af01d"
```

Comment:
i haven't verified this but if you've checked its fine,
next time you can just update the tag filter and flux will automatically find it for you and handle it.